checkstyle.xml

```diff
@@ -99,10 +104,6 @@
 			<property name="allowByTailComment" value="true"/>
 			<property name="allowNonPrintableEscapes" value="true"/>
 		</module>
-		<module name="LineLength">
```

Comment:
@kdaud  Is there any need of deleting these lines and later bringing them back?

Reply:
LineLength goes directly under Checker, not under TreeWalker because Checkstyle introduced it as a breaking change in the release notes for [September 1, 2019](https://checkstyle.org/releasenotes.html#Release_8.24).